README.md

```diff
@@ -50,20 +48,15 @@ Sailfish OS is a mobile operating system based on open source and developed by t
 
 * [Storeman](https://github.com/mentaljam/harbour-storeman) - Unofficial native
   OpenRepos.net client for Sailfish OS
-* [OwnKeepass](https://github.com/jobe-m/ownkeepass) - Password safe
-  application for Sailfish OS built on top of KeepassX
 * [Working Hours Tracker](https://wht.olpe.fi/) - Easy to use and simple
   Working Hours Tracker for SailfishOS
-* [Sailfish Apps](https://github.com/sailfishapps) - Unofficial Sailfish OS third party open source apps collection
-
-## Deploy
-* [therecipe/qt](https://github.com/therecipe/qt) - QML Cross-platform Apps with docker
```

Comment:
These exist and I am unsure why I have wanted to attempt removing them in #18?

Reply:
Lets keep it?